influxql/query_executor.go

```diff
@@ -148,13 +151,13 @@ func (e *QueryExecutor) Close() error {
 }
 
 // ExecuteQuery executes each statement within a query.
-func (e *QueryExecutor) ExecuteQuery(query *Query, database string, chunkSize int, closing chan struct{}) <-chan *Result {
+func (e *QueryExecutor) ExecuteQuery(query *Query, database string, chunkSize int, readonly bool, closing chan struct{}) <-chan *Result {
```

Comment:
Signature is getting lot of parameters (6 now).    May want to refactor this in another PR.